CoreGraphFramework/src/au/gov/asd/tac/constellation/graph/attribute/AbstractObjectAttributeDescription.java

```diff
@@ -89,39 +96,48 @@ public void setCapacity(final int capacity) {
             Arrays.fill(data, len, capacity, defaultValue);
         }
     }
-    
+
     @Override
     public String getString(final int id) {
-        return data[id] != null ? String.valueOf((T) data[id]) : null;
+        return String.valueOf((T) data[id]);
```

Comment:
This feels a bit dangerous - won't we get NPE by not checking for null?

Reply:
It's not dangerous as `String.valueOf(null)` returns `"null"`... but it does seem wrong and i'm not sure why I changed it to that. It may have just been an oversight - I'll change it back.